nxviz/plots.py

```diff
@@ -201,18 +223,27 @@ def compute_node_colors(self):
                                             )
             self.sm._A = []
 
+    def compute_group_colors(self):
+        """Computes the group colors according to node colors
+        """
```

Comment:
Minor nitpick: could you place the closing triple quote on the same line as the opening triple quote?